tools/cli/commands.js

```diff
@@ -375,7 +375,7 @@ function doRunCommand(options) {
     oplogUrl: process.env.MONGO_OPLOG_URL,
     mobileServerUrl: utils.formatUrl(parsedMobileServerUrl),
     once: options.once,
-    noReleaseCheck: options['no-release-check'],
+    noReleaseCheck: options['no-release-check'] || process.env.METEOR_NO_RELEASE_CHECK,
```

Comment:
I am fine to remove the environment variable.  I added this because sometimes we need to run more meteor commands on the development side, if we have the env var, we don't have to append the option everytime.